packages/rsu_client.rb

```diff
@@ -0,0 +1,73 @@
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./templates/packaging/usr/games/rsu-settings"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./templates/packaging/usr/games/runescape"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./templates/packaging/usr/games/update-runescape-client"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./templates/packaging/usr/local/bin/xdg-open"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./templates/protocol/install-jagex-protocol-linux"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./updater"
+
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./changelog.txt"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./install-desktop-icons"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu-settings"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/bin/rsu-query-unix"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/API/client/launch/launcher.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/Perl5lib/XML/RSSLite.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/modules/rsu/java/jre.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/modules/rsu/mains.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/modules/rsu/nvidia/optimus.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/modules/updater/download/sysdload.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/modules/updater/download/wxsysdload.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/modules/updater/gui/zenity.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/rsu-query"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./runescape"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./templates/packaging/usr/games/rsu-settings"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./templates/packaging/usr/games/runescape"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./templates/packaging/usr/games/update-runescape-client"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./templates/packaging/usr/local/bin/xdg-open"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./templates/protocol/install-jagex-protocol-linux"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./updater"
+    end
+  end
+
+  def self.build
+    system 'cpan install Archive::Extract exit'
+  end
+
+  def self.install
+    system 'mkdir', '-pv', "#{CREW_DEST_PREFIX}/share"
+    system 'cp', '-rv', 'runescape', "#{CREW_DEST_PREFIX}/share/"
+    system 'mkdir', '-pv', "#{CREW_DEST_PREFIX}/bin"
+    system 'ln', '-sv', "#{CREW_PREFIX}/share/runescape/runescape", "#{CREW_DEST_PREFIX}/bin/runescape"
```

Comment:
Is this open source or otherwise?

Reply:
This looks proprietary. If so, please close. Proprietary applications are incompatible with Chromebrew (incompatible licenses).

Reply:
Actually, the way she has it is preferable.  I would rather see a full path than a relative path when I examine a symlink.

Reply:
Not necessarily. This way, the package will be movable.

Reply:
And using the `CREW_*` variables isn't? Not sure what your point is here.

Reply:
Along with that, who the heck is going to view a symlink to a game engine? :P 
I know relative paths in symlinks are preferred in some cases elsewhere (package managers, for one).

Reply:
Who the heck is going to move things around without reinstalling?  Only you, apparently.

Reply:
Come on. The symlink will point to the same file. This way reduces the package file size and the symlink size.

Reply:
Along with that, could you please tell me why you prefer to see a full path? And why the way you like it is preferable?

Reply:
It clearly indicates the full path to the target.  Your way causes a dangling symlink if the files aren't moved relative to each other and then there would be no way to determine where the original target is located.

Reply:
All right.